.github/workflows/pull-request-comments.yml

```diff
@@ -0,0 +1,145 @@
+        with:
+          name: ${{ github.event.repository.name }}
+          path: ${{ github.event.repository.name }}
+
+      - name: Check if comment already exists
+        id: check-pr-contents
+        uses: actions/github-script@ed597411d8f924073f98dfc5c65a23a2325f34cd # v8.0.0
+        with:
+          script: |
+            // Get the pull request details.
+            const { data: pr } = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            });
+
+            // Define the string to search for.
+            const searchString = 'Test using WordPress Playground';
+
+            let foundString = false;
+
+            // Check PR body.
+            if (pr.body.includes(searchString)) {
+              foundString = true;
+            }
+
+            // Set outputs.
+            core.setOutput('string_found', foundString);
+
+            console.log(`Search string "${searchString}" found: ${foundString}`);
+
+      - name: Update PR body with a comment about testing with Playground
+        if: steps.check-pr-contents.outputs.string_found == 'false'
+        uses: actions/github-script@ed597411d8f924073f98dfc5c65a23a2325f34cd # v8.0.0
+        with:
+          script: |
+            const prNumber = context.issue.number;
+            const blueprint = {
+              preferredVersions: {
+                php: '7.4',
+                wp: 'latest',
+              },
+              steps: [
+                {
+                  step: 'mkdir',
+                  path: '/tmp/pr',
+                },
+                {
+                  step: 'writeFile',
+                  path: '/tmp/pr/pr.zip',
+                  data: {
+                    resource: 'url',
+                    url: `/plugin-proxy.php?org=WordPress&repo=ai&workflow=Build%20Plugin%20Zip&artifact=ai&pr=${prNumber}`,
+                    caption: `Downloading AI Experiments PR ${prNumber}`,
+                  },
+                },
+                {
+                  step: 'unzip',
+                  zipPath: '/tmp/pr/pr.zip',
+                  extractToPath: '/tmp/pr',
+                },
+                {
+                  step: 'installPlugin',
+                  pluginData: {
+                    resource: 'vfs',
+                    path: '/tmp/pr/ai.zip',
+                  },
+                }
+              ]
+            };
+            const blueprintUrl = `https://playground.wordpress.net/#${encodeURI(JSON.stringify(blueprint))}`;
+
+            // Get the current PR details.
+            const { data: pr } = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: prNumber
+            });
+
+            // Create the comment to append.
+            const playgroundComment = `## Test using WordPress Playground
```

Comment:
This is the exact same comment WordPress core uses but does feel pretty verbose to me. Could potentially make this much simpler